priv/repo/migrations/20170515194426_create_series.exs

```diff
@@ -0,0 +1,19 @@
+defmodule Mixdown.Repo.Migrations.CreateSeries do
+  use Ecto.Migration
+
+  def change do
+    create table(:series) do
+      add :name, :string
+      add :slug, :string
+      add :description, :string
+
+      timestamps()
+    end
+
+    alter table(:posts) do
```

Comment:
lets edit the existing posts migration file instead of this. just edit existing migrations until we are at least in beta